website/docs/flink_configuration.md

```diff
@@ -3,115 +3,179 @@ title: Flink Setup
+|----------------|--------|----------|---------------|--------------------------------------|
+| taskmanager.numberOfTaskSlots | 1 | The is the number of parallel operators or user function instances that a single TaskManager can run. We recommend setting this value > 4, and the actual value needs to be set according to the amount of data | n/a | 0.9.0 |
+| parallelism.default | 1 | The is the default parallelism used when no parallelism is specified anywhere (default: 1). For example, if the value of [`write.bucket_assign.tasks`](#parallelism-1) is not set, this value will be used | n/a | 0.9.0.|
+
+### Memory​
+The memory configuration of `JobManager` and `TaskManager` is very important for a Flink job to work smoothly. Below, we'll describe these configurations. 
+
+#### JobManager
+The JobManager handles all the instants coordination. It keeps an in-memory fs view for all the file handles on the filesystem within its embedded timeline server. We need to ensure enough memory is allocated to avoid OOM errors. The configs below allow you to allocate the necessary memory. 
+
+
+
+#### TaskManager
+The TaskManager is a container for the writing and table service tasks. For regular Parquet file flushing, we need to allocate enough memory to read and write files. At the same time, there must be enough resources for  MOR table compaction because it’s memory intensive: we need to read and merge all the log files into an output Parquet file. Below are the configs you can set for the TaskManager to allocate enough memory for these services. 
+
+| Property Name | Default  | Description | Scope | Since Version                          |
+|----------------|--------|----------|---------------|--------------------------------------|
+| jobmanager.memory.process.size | -- |This is the total process memory size for the JobManager. This includes all the memory that a JobManager JVM process consumes, consisting of Total Flink Memory, JVM Metaspace, and JVM Overhead. | n/a | 0.9.0
+| taskmanager.memory.task.heap.size | -- | This is the task heap memory size for TaskExecutors. This is the size of JVM heap memory reserved for write cache. | n/a | 0.9.0 |
+| taskmanager.memory.managed.size | -- | This is the managed memory size for TaskExecutors. This is the size of off-heap memory managed by the memory manager, reserved for sorting and RocksDB state backend. If you choose RocksDB as the state backend, you need to set this memory | n/a | 0.9.0 |
+
+#### Checkpoint​
+Checkpoint is a disaster recovery mechanism for Flink. When a job fails, users can choose to recover the job from the latest checkpoint to keep the latest data correctness. To keep the transaction integrity, we flush the memory buffer into a Hudi table for persistence during the checkpointing lifecycle. It’s important to note the Hudi transaction cannot commit the transaction without enabling the checkpoint config.
```

Comment:
Hudi transaction cannot be committed without enabling the checkpoint config.

Reply:
+1

Reply:
fixed- thanks!